schema.graphql

```diff
@@ -200,13 +200,13 @@ type Transaction @entity {
   timestamp: BigInt!
   # This is not the reverse of Mint.transaction; it is only used to
   # track incomplete mints (similar for burns and swaps)
-  mints: [Mint]!
-  burns: [Burn]!
-  swaps: [Swap]!
-  harvests: [Harvest]!
-  vests: [Vest]!
-  deposits: [Deposit]!
-  withdraws: [Withdraw]!
+  mints: [Mint!]!
+  burns: [Burn!]!
+  swaps: [Swap!]!
+  harvests: [Harvest!]!
+  vests: [Vest!]!
+  deposits: [Deposit!]!
+  withdraws: [Withdraw!]!
```

Comment:
why do we need this?

Reply:
the `!` mean that object is allow to be null/empty, I add this because It raises an error about `Mint` `Swap`... can be null